### DISCUSSION TITLE
Timed Charge Current Sliders

### DISCUSSION THREAD

**Discussion by mre150** [wills106/homeassistant-solax-modbus#429] [General]
I am using the Solis version of this integration and it mostly works great.
My only issue is with the Timed Charge Current sliders, they only allow a maximum setting of 20A.

Is there anyway to fix this issue ?

Cheers.

**Comment by wills106** [comment]
If they default to 20 your Inverter is fully added in.
What's the first 4 digits of your Serial and what should the max setting be according to your Inverter?

**Comment by mre150** [comment]
Hi
Thanks for your reply.
Inverter is  110F
It can do 62.5A of DC Charging.

**Reply by wills106** [reply]
What size is you Inverter in kW?
I have previously added in '160F' which is also 62.5A, that's a 3.6kW Inverter.

**Comment by mre150** [comment]
This one is also a 3.6kW Inverter

**Reply by wills106** [reply]
I have just released [2023.06.1b2](https://github.com/wills106/homeassistant-solax-modbus/releases/tag/2023.06.1b2)

**Reply by mre150** [reply]
Excellent, Thanks, I will give it a go.

**Reply by mre150** [reply]
Works Perfect, Thanks for the very quick fix.
